gulpfile.js

```diff
@@ -78,80 +65,82 @@ gulp.task('unit-tests', function () {
  * 
  * 1. clean-dev
  * 2. build-dev
- * 3. inject-dev
+ * 3. prepare-dev
  */
 
 // Clean all the output directories
 gulp.task('clean-dev', function( cb ) {
-	del([
-	     './target/dev/js',
-	     './target/dev/css',
-	     './target/dev/templates',
-	     './target/dev/img',
-	     './target/dev/index.html'
-	], cb);
+  del([
+       './target/dev/js',
+       './target/dev/css',
+       './target/dev/templates',
+       './target/dev/img',
+       './target/dev/index.html'
+       ], cb);
 });
 
 // Shortcut for all the DEV tasks
-gulp.task('build-dev', [ 'clean-dev' ], buildDevDirectory )
-gulp.task('inject-dev', [ 'build-dev' ], injectScriptsInDev )
```

Comment:
The `inject-dev` task is referenced in the `build.xml` file of the roboconf-dm-web-administration project (roboconf-platform repository). Merge will fail the build until the Ant build file is updated with a new task name (which one?).